documentation/overview.md

```diff
@@ -260,41 +270,38 @@ df.write.format("com.microsoft.cdm")
  .option("entityDefinitionContainer", "cdmmodels")
  .option("entityDefinitionModelRoot", "core")
  .option("entityDefinition", "/Contacts/Person.cdm.json/Person")
- .option("appId", appid)
- .option("appKey", appkey)
- .option("tenantId", tenantid)
  .mode(SaveMode.Overwrite)
  .save()
 ```
+#### Explicit Write - using an entity defined in the CDM GitHub
 
-#### Read
-
-This code reads the Person entity from the CDM folder with manifest at [https://mystorage.dfs.core.windows.net/cdmdata/contacts/root.manifest.cdm.json](https://mystorage.dfs.core.windows.net/cdmdata/contacts/root.manifest.cdm.json)
+This code writes the dataframe _df_ to a CDM folder with the manifest at [https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json](https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json) and a sub-manifest containing the TeamMembership entity, created in a TeamMembership subdirectory. TeamMembership data is written as CSV files (by default) that overwrite any existing data files. The entity definition is retrieved from the CDM GitHub repo, at:
+[https://github.com/microsoft/CDM/tree/master/schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership](https://github.com/microsoft/CDM/tree/master/schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership)
 
 ```scala
-val df = spark.read.format("com.microsoft.cdm")
+df.write.format("com.microsoft.cdm")
  .option("storage", "mystorage.dfs.core.windows.net")
  .option("container", "cdmdata")
- .option("manifest", "/contacts/root.manifest.cdm.json")
- .option("entity", "Person")
- .option("useCDMGithub", true)
- .option("appId", appid)
- .option("appKey", appkey)
- .option("tenantId", tenantid)
- .load()
+ .option("manifest", "Teams/root.manifest.cdm.json")
```

Comment:
Remove container and manifest option and replace with manifestPath - cdmdata/Teams/root.manifest.cdm.json